src/NHSD.BuyingCatalogue.Identity.Api/Settings/IdentityResourceSetting.cs

```diff
@@ -1,5 +1,5 @@
-﻿using IdentityServer4.Models;
-using NHSD.BuyingCatalogue.Identity.Api.Constants;
+﻿using BuyingCatalogue.Identity.Common.Constants;
```

Comment:
Any reason why the NHSD prefix is missing from the using statement?

Reply:
Yeah I missed it in my initial checkin, it's added now